index.html

```diff
@@ -307,7 +313,7 @@ <h3>Proofs</h3>
           <dt><dfn>verificationMethod</dfn><dt>
           <dd>
 <em>Required</em>. A set of parameters required to independently verify the
-proof, such as an identifier for a public/private key pair that would be used in
+proof, such as an identifier for key material that would be used in
```

Comment:
```suggestion
proof, such as an identifier for cryptographic material that would be used in
```

We might also want to crib from https://www.w3.org/TR/did-core/#dfn-verification-method